srcCpp/perftest_publisher.cxx

```diff
@@ -2212,6 +2212,23 @@ inline void perftest_cpp::SetTimeout(unsigned int executionTimeInSeconds,
       #ifdef RTI_WIN32
         CreateTimerQueueTimer(&_hTimer, _hTimerQueue, (WAITORTIMERCALLBACK)Timeout_scan,
                 NULL , executionTimeInSeconds * 1000, 0, 0);
+      #elif defined RTI_INTIME
+            HANDLE  hTimer;
+            hTimer = RtCreateTimer(NULL, 0, Timeout_scan, NULL, 
+                        RT_PRIORITY_MAX, CLOCK_FASTEST);
+            if (NULL == hTimer) {
+                    printf("Can not create the timer!!\n");
+                    exit(1);
+            }
+            LARGE_INTEGER timeout;
+            // set timer absolute
+            if (!RtGetClockTime(0, &timeout))
+                printf("Cannot get clock time!!\n");
+            timeout.QuadPart += executionTimeInSeconds*1000*10000;   /*Time in 100 nano seconds */
```

Comment:
timeout.QuadPart += executionTimeInSeconds * 1000 * 10000;   /*Time in 100 nano seconds */

Reply:
I have checked in new changes, but i dont know how to apply those changes to this pull request.
Please can you update the pull request with new changes?